### PR TITLE
Nexus Mutual - deprecate old spells & convert v1 views to static tables

### PR DESCRIPTION
**Pull request by tomfutago**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:**
  - deprecate old unused spells: `capital_pool_eth_daily_transaction_summary` & `nexusmutual_ethereum_capital_pool_eth_daily_transaction_summary`
  - convert `claims_v1` & `covers_v1` views to static tables - no new records expected (assuming materialized as 'table' and 'static' tag will do the trick here?)

---

### CODE REVIEW

File: dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/claims/nexusmutual_ethereum_claims_v1.sql [hunk 1]
```diff
@@ -2,12 +2,13 @@
   config(
     schema = 'nexusmutual_ethereum',
     alias = 'claims_v1',
-    materialized = 'view',
+    materialized = 'table',
+    tags = ['static'],
```
Comment (jeff-dude) [inline dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/claims/nexusmutual_ethereum_claims_v1.sql:6]:
```suggestion
```

Comment (jeff-dude) [reply dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/claims/nexusmutual_ethereum_claims_v1.sql:6]: you should remove static, as it reads from tables. we use `static` when the spell is just hardcoded strings and no table reads

File: dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/covers/nexusmutual_ethereum_covers_v1.sql [hunk 2]
```diff
@@ -2,11 +2,9 @@
   config(
     schema = 'nexusmutual_ethereum',
     alias = 'covers_v1',
-    materialized = 'incremental',
-    file_format = 'delta',
-    incremental_strategy = 'merge',
+    materialized = 'table',
+    tags = ['static'],
```
Comment (jeff-dude) [inline dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/covers/nexusmutual_ethereum_covers_v1.sql:6]:
```suggestion
```

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/claims/nexusmutual_ethereum_claims_v1.sql (hunk 1 above)
Comment (tomfutago) [reply dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/claims/nexusmutual_ethereum_claims_v1.sql:6]: ah cool, so if all I want is to have this data as a table which is loaded once and just sits there - `materialized = 'table'` is all I need?

Comment (jeff-dude) [reply dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/claims/nexusmutual_ethereum_claims_v1.sql:6]: we have removed usage of `static` tag in our prod orchestration atm. you could actually leave in then if you want, so if ever in the future we want to revisit adding, we would know this model can be static

Comment (tomfutago) [reply dbt_subprojects/daily_spellbook/models/nexusmutual/ethereum/claims/nexusmutual_ethereum_claims_v1.sql:6]: good to know, let's leave it it then

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**